examples/fips/README.md

```diff
@@ -0,0 +1,83 @@
+# FIPS Compliance
+
+We are testing and making this client FIPS compliant using OpenSSL 3.0. Use OpenSSL 3.0 to use this client in FIPS compliant mode. Older versions of OpenSSL might work in similar way but we have not verified that.
+
+## Communication between client and Kafka cluster
+
+### Install from prebuilt wheels
+
+If this client is installed using `pip`, it comes with librdkafka in which OpenSSL 3.0 is statically linked. This client can communicates with Kafka cluster with FIPS approved algorithms using FIPS provider provided by OpenSSL. Steps to enable FIPS provider is mentioned in section [Enabling FIPS provider](#enabling-fips-provider).
```

Comment:
```suggestion
If you install this client using `pip`, you get the client with librdkafka with a static link to OpenSSL 3.0. To enable this client to communicate with the Kafka cluster using the OpenSSL FIPS provider and FIPS-approved algorithms, you must enable the FIPS provider. You can find steps to enable the FIPS provider in section [Enabling FIPS provider](#enabling-fips-provider).
```
I'm guessing that the user installs and then needs to enable? If not, let me know and I can rework this section.

Reply:
Used your suggestion with some changes. Please check.